### PR TITLE
updated to align with current spec main branch

### LINKED ISSUE #121: Align SDK with specification 0.7
**What would you like to be added**:

Align SDK with specification 0.7, current [specification main branch](https://github.com/serverlessworkflow/specification/)

**Why is this needed**:

### PR DESCRIPTION
**Pull request by antmendoza**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
updated to align with current spec main branch

closes #121 

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: src/lib/definitions/action.ts [hunk 1]
```diff
@@ -16,26 +16,42 @@
 import { Actiondatafilter } from './actiondatafilter';
 import { Eventref } from './eventref';
 import { Functionref } from './functionref';
-import { overwriteActionDataFilterValue, overwriteEventRefValue, overwriteFunctionRefValue } from './utils';
+import {
+  normalizeSubFlowRefIfObject,
```
Comment (antmendoza) [inline src/lib/definitions/action.ts:20]: Can you think in a better name for this methods ?

**Comment by tsurdilo** [comment]
@antmendoza thanks!! +100

**Review by tsurdilo — APPROVED** [review]

**Comment by JBBianchi** [comment]
Great job @antmendoza !

The only thing I see right now is the missing update for the README for the Release/Conformance to spec version.

**Comment by tsurdilo** [comment]
please do a separate release if possible as curently we are on 0.6 right?
also i would wait a little as some more changes will come in for 0.7 in next weeks

**Comment by JBBianchi** [comment]
I'd suggest a release as 2.0.0-next.0 with a @next tag.

**Comment by antmendoza** [comment]
ok @JBBianchi , I am going to have a look 

thank you!!

**Review by JBBianchi — APPROVED** [review]

**Pull request merged**